channel-api/src/main/java/com/synopys/integration/alert/channel/api/issue/model/IssueTransitionType.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * channel-api
+ *
+ * Copyright (c) 2021 Synopsys, Inc.
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package com.synopys.integration.alert.channel.api.issue.model;
+
+public enum IssueTransitionType {
+    REOPEN,
```

Comment:
I think OPEN would make more sense here. If we're thinking in terms of a general Issue manager, we open and resolve Issues. REOPEN to me implies we don't OPEN in the first place

Reply:
FWIW, this only represents the operations we would perform on _existing_ issues, not new issues.